nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -378,7 +378,10 @@ private ClassTree findEnclosingLocalOrAnonymousClass(ClassTree classTree) {
   @Override
   public TransferResult<Nullness, NullnessStore> visitNullChk(
       NullChkNode nullChkNode, TransferInput<Nullness, NullnessStore> input) {
-    throw new RuntimeException("we should never see this");
+    SubNodeValues values = values(input);
+    Nullness nullness =
+        hasPrimitiveType(nullChkNode) ? NONNULL : values.valueOfSubNode(nullChkNode.getOperand());
```

Comment:
I don't think the compiler will ever insert a `NullChk` around a node of primitive type, but we can leave the check out of paranoia 😄